### PR TITLE
Rename tiles behavior and fields to blocks, migration step

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@tisto whenever you can, merge this and cut a release. Thx.

File: docs/source/tiles.rst [hunk 1]
```diff
@@ -3,9 +3,6 @@ Tiles
 
 .. note::
   The tiles endpoint currently match only partially (the GET endpoints) the default Plone implementation.
-  The serialization of tiles didn't match the Mosaic (and plone.app.blocks) implementation and it's done to
```
Comment (tisto) [inline docs/source/tiles.rst]: @sneridagh shouldn't we leave a note here that at some point we are going to remove that endpoint? As far as we know nobody ever used that endpoint anyways.

**Pull request merged**